tempesta_fw/sock_srv.c

```diff
@@ -2035,7 +2056,10 @@ tfw_cfgop_update_sg_srv_list(TfwCfgSrvGroup *sg_cfg)
 
 	TFW_DBG2("Update server list for group '%s'\n", sg_cfg->orig_sg->name);
 
-	__tfw_sg_for_each_srv(sg_cfg->orig_sg, __tfw_cfgop_update_sg_srv_list, sg_cfg);
+	if ((r = __tfw_sg_for_each_srv(sg_cfg->orig_sg,
```

Comment:
Line is too log, is was also warned a couple of time that the next formatting is better:
```
r = __tfw_sg_for_each_srv(sg_cfg->orig_sg, __tfw_cfgop_update_sg_srv_list,
                          sg_cfg);
if (r)
    return r;
```

Reply:
Corrected.